Resources/Prototypes/_Pirate/Entities/Mobs/Customization/Markings/reptilian_marking.yml

```
@@ -0,0 +1,63 @@
+    state: eyebrowsunibrow
+
+- type: marking
+  id: ReptilianTattooEyeRight
+  bodyPart: Eyes
+  markingCategory: Face
+  speciesRestriction: [Reptilian] # Delta V - Felinid, Oni, Harpy
+  coloring:
+    default:
+      type:
+        !type:EyeColoring
+          negative: true
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)